### ISSUE TITLE
fix: Return detached attributions without the gradient information

### ISSUE THREAD

**Issue by WolodjaZ** [xai4space/meteors#150]
### Checklist

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I am sure that this is a bug in Meteors rather than my code.
- [X] The bug is not resolved by updating to the latest stable version of Meteors.
- [X] I included a link to the documentation page I am referring to (if applicable).

### Reproduction steps

```
ig_attr = ig.attribute(
    hsi_0,
    target=0,
    baseline=0.0,
    return_convergence_delta=True,
    n_steps=50,
)
assert ig_attr.attributes.requires_grad == False
```
